amr-wind/wind_energy/actuator/FLLCOp.H

```diff
@@ -191,12 +193,27 @@ struct FLLCOp
             const auto vmag =
                 std::max(vs::mag(vel), vs::DTraits<amrex::Real>::eps());
             const auto vmag2 = vmag * vmag;
-
+            fllc.vel_rel[ip] = vel;
             const auto fv = force & vel;
 
             G[ip] = (force - vel * fv / vmag2) / dr;
         }
 
+        // Get the non-uniform distribution of force
```

Comment:
I think this needs conditionals around it if the non-uniform spacing is optional.  It seems like the non-uniform points wouldn't be populated.

Reply:
Added conditional in latest commit.